src/components/Learn-Components/Course-Overview-page/index.js

```diff
@@ -1,24 +1,30 @@
 import React from "react";
 import { CourseOverviewWrapper } from "./course-overview.styles";
-import {  Row, Col } from "../../../reusecore/Layout";
+import { Row, Col } from "../../../reusecore/Layout";
 import Image from "../../image";
+import Course1 from "../../../assets/images/learn/sercice-mesh-course1.png";
 
 import { Link } from "gatsby";
-import { IoDocumentTextOutline, IoVideocam,IoChevronBackOutline } from "react-icons/io5";
+
+import {
+  IoDocumentTextOutline,
+  IoVideocam,
+  IoChevronBackOutline,
+} from "react-icons/io5";
```

Comment:
These icons are not used anywhere, it can be removed.

Reply:
They are used below I guess.

Reply:
Ahh...right. But, do remove the unused images @Anita-ihuman .